docs/topics/assembly_building-and-deploying-the-secured-booster-to-openshiftlocal.adoc

```diff
@@ -1,6 +1,6 @@
 
-[#building-and-deploying-the-secured-booster-to-openshiftlocal]
-= Building and deploying the {name-mission-secured} booster to {OpenShiftLocal}
+[#deploying-the-secured-booster-using-openshiftlocal]
+=  Deploying the {name-mission-secured} booster using {OpenShiftLocal}
```

Comment:
Same as above -- I think "to" is more accurate than "using".

Reply:
@Ladicek will take your suggestion and change using->to

Reply:
Still uses "using" here :-)